frontend/src/lib/utils.js

```diff
@@ -1,27 +1,24 @@
 import newGithubIssueUrl from 'new-github-issue-url'
 
-export const issueUrl = ( { id, gurmukhi } ) => newGithubIssueUrl( {
+export const issueUrl = ( { id, gurmukhi, page, source } ) => newGithubIssueUrl( {
   user: 'ShabadOS',
   repo: 'Database',
   assignee: 'Sarabveer',
   labels: [ 'correction' ],
-  title: `${id} - Gurbani Dian Laga Matra Di Vilakhanta`,
+  title: `${id}`,
   body: `
-  > Use the preview tab to see the example given below. Edit the relevant information and preview the changes before submitting. You may delete these instructions.
-  
-  | Key              | Value |
-  | ---------------- | -------- |
-  | ID                 | ${id} |
-  | Shabad OS  | \`${gurmukhi}\` |
-  | Correction    | \`goibMd vwlu\` ≠ \`goibMdvwlu\`  |
-  
-  Gurbani Dian Laga Matra Di Vilakhanta (3rd Edition, November 2003)
-  
-  Page 366
-  ![](https://i.imgur.com/GcVYt8h.jpg)
-  
-  Page 367   
-  ![](https://i.imgur.com/0z7Q4gd.jpg)
-  
-  `,
+> Use the preview tab to see the example given below. Edit the relevant information and preview the changes before submitting. You may delete these instructions.
+
+| Key | Value |
+| --- | --- |
+| ID | ${id} |
+| Source | ${source} |
+| Page | ${page} |
+| Line | \`\`\`${gurmukhi}\`\`\` |
```

Comment:
Too much escaping here.

This will render as \``this is gurmukhi`\`, instead of `this is gurmukhi`

Reply:
There needs to be three backticks. Gurbani can have a single backtick (used as a character) which would cancel out the first at the start.

Reply:
Other option is to not show it in code (it's really just for style). Could be shown without any backticks at all in the table